sw/device/lib/dif/dif_clkmgr.h

```diff
@@ -148,6 +148,20 @@ dif_result_t dif_clkmgr_hintable_clock_get_hint(
     const dif_clkmgr_t *clkmgr, dif_clkmgr_hintable_clock_t clock,
     dif_toggle_t *state);
 
+/**
+ * Enable chip to use the external clock.
+ *
+ * The input arg is_low_speed indicates what speed the external clocks is.
+ * High speed - external clock is close to nominal speeds (e.g. external clock
+ * is 96MHz and nominal frequency is 96MHz-100MHz). Low speed - external clock
+ * is half of nominal speeds (e.g. external clock is 48MHz and nominal frequency
+ * is 96MHz-100MHz).
+ *
```

Comment:
While you have a description of the input arg, you want to make sure you format it using the doxygen syntax like the other DIF comment descriptions (see [here](https://github.com/lowRISC/opentitan/blob/4d7a648fcc5a51a4c89dc8df1fb8ce22a7fae728/sw/device/lib/dif/dif_clkmgr.h#L45-L47)), since we use doxygen to autogen the [DIF API documentation](https://docs.opentitan.org/sw/apis/dif__clkmgr_8h.html).

Reply:
No wonder there are special format in the comments of other dif functions. Thanks for letting me know.
Updated.